package.json

```diff
@@ -81,6 +84,9 @@
   "babel": {
     "presets": [
       "react"
+    ],
+    "plugins": [
```

Comment:
@rauchg: @pcalves:

The way to use "transform-react-remove-prop-types" in this case is to also override the default mode:
```json
[
  "transform-react-remove-prop-types", {
    "mode": "wrap"
  }
]
```

This should change `propTypes` in source:
```js
class MyComponent extends Component {
  static propTypes = {
    foo: PropTypes.string
  };
}
function MyFunctionalComponent ({ bar }) {
}
MyFunctionalComponent.propTypes = {
  bar: PropTypes.string
}
```

To this:
```js
class MyComponent extends Component {
}
if (process.env.NODE_ENV !== "production") {
  MyComponent.propTypes = {
    foo: PropTypes.string
  };
}
function MyFunctionalComponent ({ bar }) {
}
if (process.env.NODE_ENV !== "production") {
  MyFunctionalComponent.propTypes = {
    bar: PropTypes.string
  };
}
```

The default mode ("remove") is good for application code. "wrap" is meant for library code. It leaves `propTypes` in place to be later removed.